style.css

```diff
@@ -0,0 +1,118 @@
+@import "https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap";
+
+:root {
+  --main-bg-color: #fff;
+  --normal-text-color: #344563;
+  --gray-background-color: #fff;
+  --container-bg: #e5e5e5;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+  overflow-y: auto;
+  outline: none;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+li {
+  list-style: none;
+}
+
+.container {
+  width: 380px;
+  margin: 0 auto;
+  border: 1px solid #7f8cff;
+  background-color: var(--container-bg);
+}
```

Comment:
- [ ] **Good Job so far** 👍👍 However, I think it would be better to avoid using a ```fixed width``` for your container as this may cause issues with **responsiveness**, especially on smaller screens or devices with different screen resolutions. 😉😉

    As this is a **mobile-first setup**, the mobile version should be responsive until the desktop screen size, you may try to use ```100vw```, or you set your width to 100% to keep it responsive 👍 👍 

## Your design
![image](https://user-images.githubusercontent.com/102757126/211598786-0e14e0cd-06f4-40d5-8328-b8579592951d.png)